### PR TITLE
AAE-5586 make password strong and random

### PR DESCRIPTION
**Pull request by alxgomz**
even though they shouldn't be used (as authentication should remain confimed at keycoak level) we don't want to have simple or hardcoded passwords.
ref AAE-5586

### CODE REVIEW

File: helm/alfresco-process-infrastructure/setup_acs.sh [hunk 1]
```diff
@@ -7,10 +7,10 @@ add_user_to_group "service-account-storage-service" "ALFRESCO_ADMINISTRATORS"
 
 if [[ "$LOAD_TEST_DATA" == "true" ]]
 then
-  create_user "admin.adf" "admin.adf@alfresco.com"
+	create_user "admin.adf" "admin.adf@alfresco.com" $(generate_alfpwd)
```
Comment (mteodori) [inline helm/alfresco-process-infrastructure/setup_acs.sh:10]: we should do for the service-account user for PaaS, test data is for testing and can stay with default

**Review by mteodori — CHANGES_REQUESTED** [review]

**Review by TheOutsSeeker — APPROVED** [review]
LGTM

File: helm/alfresco-process-infrastructure/setup_acs.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 source $(dirname "$0")/functions_acs.sh
 
-create_user "service-account-storage-service"
+create_user "service-account-storage-service" "sass@activiti.org" $(generate_alfpwd)
```
Comment (mteodori) [inline helm/alfresco-process-infrastructure/setup_acs.sh:5]: use service-account-storage-service@example.com as password as that is an official dummy domain to use

**Review by mteodori — CHANGES_REQUESTED** [review]

File: helm/alfresco-process-infrastructure/setup_acs.sh (hunk 2 above)
Comment (alxgomz) [reply helm/alfresco-process-infrastructure/setup_acs.sh]: done

**Review by mteodori — APPROVED** [review]

**Pull request merged**